### ISSUE TITLE
 Description of floats

### ISSUE THREAD

**Issue by kathriwa** [arXiv/html_feedback#2795]
### Description

The \Description{...} in float-elements should not be visible in the text of the website but should instead be treated as alt-text in the html.

### (Optional:) Please add any files, screenshots, or other information here.

<img width="875" alt="screenshot of a table with a visible description" src="https://github.com/user-attachments/assets/a1510b28-1302-4f88-98d9-f66a606815f8">

### (Required) What is this issue most closely related to? Select one.

Figures

### Internal issue ID

4808b3c5-0d18-4ae9-bbf2-47dfd4525f41

### Paper URL

https://arxiv.org/html/2412.14193v2

### Browser

Firefox/133.0

### Device Type

Desktop

**Comment by dginev** [comment]
Bookkeeping comment: This is pending the upgrades from https://github.com/brucemiller/LaTeXML/pull/2666

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
